dealership.rb

```diff
@@ -1,29 +1,58 @@
+require 'csv'
+
+module CarLoader
+  def self.get_cars_from_csv(filepath)
+    @cars = CSV.read(filepath, { :headers => true, :header_converters => :symbol}).map { |car_object| Car.new(car_object) }
+  end
+
+  def save
+    CSV.open('inventory.csv',"w") do |csv|
+      @tasks.each do |task|
```

Comment:
It looks like you have copy and pasted this code from the todo app. I want you to copy and paste less, and read, parse, and understand example code more. You need to understand an example, put it away and then rewrite it for your own context.

You don't have access to any `@tasks` variable here. You need to pass an argument into this method. The method should be expecting an array of car objects. You should then push the headers into the CSV file. You should then iterate over those car objects and push the relevant data (as an array of strings) into the the file.